### PR TITLE
Change JUL Logging to SLF4J

### LINKED ISSUE #315: Replace Java Util Logging with SLF4J
I have been trying to configure easybatch with `spring-boot` and it have been a  nightmare to get logs redirecting to either log-back or log4j.  Even after being able to configure system fail with class-path errors. Spring boot guys mention in their documentation 

```
There are known classloading issues with Java Util Logging that cause problems when running from an ‘executable jar’. We recommend that you avoid it if at all possible.
```

I know that `we want the framework's core to have zero dependencies` but dont  you think is reasonable to think about this again? 

Note that was already mention
https://github.com/j-easy/easy-batch/issues/125

### PR DESCRIPTION
**Pull request by MALPI**
Fixes #315.

I tried to map all levels according to https://www.slf4j.org/apidocs/org/slf4j/bridge/SLF4JBridgeHandler.html

### CODE REVIEW

**Comment by benas** [comment]
Hi @MALPI 

Thank you very much for this PR! I really appreciate your effort!
I added a few comments which are really minor, you did already the hard work 👍 

Please do not hesitate also to add your name to the contributors list in the readme.

Just a quick information, the build on windows is ko: https://ci.appveyor.com/project/benas/easy-batch/build/1.0.43

I'm not sure if it is related to your changes, but can you please take a look just in case?

Many thanks upfront!
Kr
Mahmoud

File: easybatch-xml/src/main/java/org/easybatch/xml/XmlRecordReader.java [hunk 1]
```diff
@@ -141,7 +141,7 @@ private boolean nextTagIsRootElementEnd() throws XMLStreamException {
                 xmlEventReader.peek().asEndElement().getName().getLocalPart().equalsIgnoreCase(rootElementName);
     }
 
-    private void writeEndElement(StringBuilder stringBuilder, XMLEvent xmlEvent) throws XMLStreamException {
+    private void writeEndElement(StringBuilder stringBuilder, XMLEvent xmlEvent) {
```
[the remark this answers is not in the document]
Comment (MALPI) [reply easybatch-xml/src/main/java/org/easybatch/xml/XmlRecordReader.java]: Filed #317 for it.

**Comment by benas** [comment]
Hi @MALPI 

Fantastic! Thank you for all these updates 👍 

I've merged your PR which will be part of v5.3. Keep tuned.

Best regards,
Mahmoud

**Comment by benas** [comment]
@MALPI I added you to the awesome contributors list 7c83757d56f06712a9526c5b487d8453b7859311 !

**Comment by MALPI** [comment]
When are you planning to release the next iteration?

**Comment by benas** [comment]
There is no enough material [done](https://github.com/j-easy/easy-batch/milestone/13) for v5.3 yet.. So I'm not sure when to do it.

You can still use the snapshot in the meantime. It's already deployed to maven central with your changes.

**Comment by MALPI** [comment]
Hey @benas can you tell me where exactly I can find the snapshots? Btw it's year since I made the changes what about releasing a bugfix version or something similar?

Best

**Comment by benas** [comment]
The latest snapshot can be found in sonatype's snapshot repo: https://oss.sonatype.org/content/repositories/snapshots/org/easybatch/

Sorry for not being active here for a while, I'm currently releasing Easy Random and Easy Rules (planned for April 28th) then I will give Easy Batch the love it deserves.

Thank you for your comprehension!

**Pull request merged**